pmm/aws_iam_roles_policies.tf

```diff
@@ -0,0 +1,57 @@
+# These roles and policies should be created first as "global" resources
```

Comment:
@utdrmac How are these created then in the first place? Is there a separate command we need to run once per region?

Reply:
added resources into pmm/ dir